diff --git a/base/BUILD.gn b/base/BUILD.gn
index feab31731067e946a55ac2bd13fa0f600659297d..67cda38d98a7ef56a0f15e44d5003894449e7343 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4977,6 +4977,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5643,6 +5644,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
