diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
index 1c5eed567b3660e20b80e5ffcadfa23efd89f3ef..e02bf6674eb1d281644f8a5d5365e82efff2a8b4 100644
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1706,6 +1706,7 @@ void RegisterLocalState(PrefRegistrySimple* registry) {
       false);
 
   // This is intentionally last.
+  brave::RegisterLocalStatePrefs(registry);
   RegisterLocalStatePrefsForMigration(registry);
 }
 
@@ -2215,6 +2216,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry,
 #endif  // BUILDFLAG(IS_ANDROID)
 
   registry->RegisterBooleanPref(prefs::kStaticStorageQuotaEnabled, false);
+  brave::RegisterProfilePrefs(registry);
 }
 
 void RegisterUserProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -2584,6 +2586,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
   profile_prefs->ClearPref(kObsoletePasswordsUseUPMLocalAndSeparateStores);
   profile_prefs->ClearPref(kObsoleteEmptyProfileStoreLoginDatabase);
   profile_prefs->ClearPref(kObsoleteUpmAutoExportCsvNeedsDeletion);
+#if 0
   base::DeleteFile(profile_path.Append(FILE_PATH_LITERAL("Login Data")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account")));
@@ -2591,6 +2594,7 @@ void MigrateObsoleteProfilePrefs(PrefService* profile_prefs,
       profile_path.Append(FILE_PATH_LITERAL("Login Data-journal")));
   base::DeleteFile(
       profile_path.Append(FILE_PATH_LITERAL("Login Data For Account-journal")));
+#endif
 #endif  // BUILDFLAG(IS_ANDROID)
 
   // Added 09/2025.
