diff --git a/chrome/updater/mac/.install.sh b/chrome/updater/mac/.install.sh
index 3b1874d09fca0cf88e361927dfea7c4273790030..dcb6aa3dea97f47a25c2f282d887d3653b143d31 100755
--- a/chrome/updater/mac/.install.sh
+++ b/chrome/updater/mac/.install.sh
@@ -338,6 +338,7 @@ main() {
 
   # update_app is the path to the new version of the .app.
   local update_app="${update_dmg_mount_point}/${APP_DIR}"
+  update_app="${update_dmg_mount_point}/${PRODUCT_NAME}.app" # The value in the previous line is wrong when install directory has a non-standard name. Eg. Brave.app instead of Brave Browser.app.
   note "update_app = ${update_app}"
 
   # Make sure that it's an absolute path.
