diff --git a/chrome/browser/startup_data.cc b/chrome/browser/startup_data.cc
index 3631126b2f5e78aa719f448ed5cc5e9d46e0f029..aebad1eb62d5f9b9ee3536ea8824cc41cc0ae000 100644
--- a/chrome/browser/startup_data.cc
+++ b/chrome/browser/startup_data.cc
@@ -223,6 +223,7 @@ void StartupData::CreateServicesInternal() {
       true /* force_immediate_policy_load */, io_task_runner,
       base::BindRepeating(&content::GetNetworkConnectionTracker));
 
+  BRAVE_STARTUP_DATA_PRE_PROFILE_POLICY_CONNECTOR
   profile_policy_connector_ = policy::CreateAndInitProfilePolicyConnector(
       schema_registry_service_->registry(),
       static_cast<policy::ChromeBrowserPolicyConnector*>(
@@ -230,6 +231,7 @@ void StartupData::CreateServicesInternal() {
       user_cloud_policy_manager_.get(),
       user_cloud_policy_manager_->core()->store(),
       true /* force_immediate_policy_load*/, nullptr /* user */);
+  BRAVE_STARTUP_DATA_POST_PROFILE_POLICY_CONNECTOR
 
   RegisterProfilePrefs(false /* is_signin_profile */,
                        chrome_feature_list_creator()->actual_locale(),
