diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 01ffea69e2b98588814d84370426c7fb761e6f83..62bbb65b388a9d4392971d3a04f37c4caeba597e 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -574,6 +574,7 @@ void ToolbarView::Init() {
   }
 
   if (!features::IsWebUIAvatarButtonEnabled()) {
+    BRAVE_TOOLBAR_VIEW_INIT
     avatar_ =
         AddChildView(std::make_unique<AvatarToolbarButton>(browser_view_));
     bool show_avatar_toolbar_button = true;
