diff --git a/chrome/browser/ui/views/page_action/page_action_view.cc b/chrome/browser/ui/views/page_action/page_action_view.cc
index d889a084b2b40c4140ff01ae050c19a343cce3be..8eee7dbe248737b2294dbf4557f0f123fe1ec898 100644
--- a/chrome/browser/ui/views/page_action/page_action_view.cc
+++ b/chrome/browser/ui/views/page_action/page_action_view.cc
@@ -167,7 +167,7 @@ void PageActionView::OnPageActionModelChanged(
   if (!visible) {
     ResetSlideAnimation(/*show=*/false);
     NotifyIsChipShowingChange();
-  } else if (model.ShouldShowSuggestionChip()) {
+  } else if (model.ShouldShowSuggestionChip() || model.GetAlwaysShowLabel()) {
     if (model.GetShouldAnimateChipIn()) {
       AnimateIn(/*string_id=*/std::nullopt);
     } else {
