diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 612fb8c65760d03043fd0b3d74410838f60ebf5b..daaefd5a3b6029aed4f65e08b67578e245d933c0 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1558,6 +1558,7 @@ source_set("extensions") {
       "api/tabs/app_window_helper.h",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 static_library("test_support") {
