diff --git a/chrome/browser/ui/page_actions/page_action_controller.h b/chrome/browser/ui/page_actions/page_action_controller.h
index 0135daf089d8564440c30ca17fd5c31c84e335e7..0457c9eae6d136545a1a585fbe86028b1c64328c 100644
--- a/chrome/browser/ui/page_actions/page_action_controller.h
+++ b/chrome/browser/ui/page_actions/page_action_controller.h
@@ -167,6 +167,27 @@ class PageActionController {
 
   virtual ~PageActionController() = default;
 
+  // Force the label to be shown regardless of the available space
+  virtual void SetAlwaysShowLabel(actions::ActionId action_id,
+                                  bool always_show) = 0;
+
+  // Override the colors of the chip
+  virtual void OverrideChipColors(
+      actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) = 0;
+  virtual void ClearOverrideChipColors(actions::ActionId action_id) = 0;
+
+  // Override the height of the PageActionView
+  virtual void SetOverrideHeight(actions::ActionId action_id,
+                                 int height) = 0;
+  virtual void ClearOverrideHeight(actions::ActionId action_id) = 0;
+
+  // When set, `PageActionView::IsTriggerableEvent` uses this callback instead of
+  // the default logic. Pass nullopt to clear.
+  virtual void SetOverrideTriggerableEvent(actions::ActionId action_id, 
+                                           std::optional<int> event_flags) = 0;
+
   // Requests that the page action be shown or hidden.
   virtual void Show(actions::ActionId action_id) = 0;
   virtual void Hide(actions::ActionId action_id) = 0;
@@ -297,6 +318,18 @@ class PageActionControllerImpl : public PageActionController,
   PageActionControllerImpl& operator=(const PageActionControllerImpl&) = delete;
   ~PageActionControllerImpl() override;
 
+  void SetAlwaysShowLabel(actions::ActionId action_id,
+                          bool always_show) override;
+  void OverrideChipColors(actions::ActionId action_id,
+      std::optional<SkColor> override_background_color,
+      std::optional<SkColor> override_foreground_color) override;
+  void ClearOverrideChipColors(actions::ActionId action_id) override;
+  void SetOverrideHeight(actions::ActionId action_id,
+                         int height) override;
+  void ClearOverrideHeight(actions::ActionId action_id) override;
+  void SetOverrideTriggerableEvent(actions::ActionId action_id,
+                                   std::optional<int> event_flags) override;
+
   void Initialize(
       tabs::TabInterface& tab_interface,
       const std::vector<actions::ActionId>& action_ids,
