diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index b6232eecd46678ebfa8151ec449a79cf506a4003..953ea4dc9affa49cd01907f1cf1d5675cc162d67 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1300,6 +1300,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
         base::MakeRefCounted<SearchProvider>(provider_client_.get(), this);
     search_provider_ = search_provider.get();
     providers_.push_back(std::move(search_provider));
+    BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1572,6 +1573,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type,
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
 
   const bool is_lens_enabled = autocomplete_provider_client()->IsLensEnabled();
