diff --git a/components/autofill/core/browser/ui/autofill_external_delegate.cc b/components/autofill/core/browser/ui/autofill_external_delegate.cc
index 471a06eebbcb0769770bcce5dadead3824d7230f..7630ca26b4e0091c34d95ba8e7085956dbcfac29 100644
--- a/components/autofill/core/browser/ui/autofill_external_delegate.cc
+++ b/components/autofill/core/browser/ui/autofill_external_delegate.cc
@@ -338,6 +338,7 @@ void AutofillExternalDelegate::AttemptToDisplayAutofillSuggestions(
   // updated to match.
   InsertDataListValues(suggestions);
 
+  BRAVE_AUTOFILL_EXTERNAL_DELEGATE_ATTEMPT_TO_DISPLAY_AUTOFILL_SUGGESTIONS
   // TODO(crbug.com/362630793): Try to eliminate this state. The controller
   // should be the one that knows about what suggestions were shown and passes
   // it on, not AED.
