diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 9ed93f1b8c497f51f559f53a1ac668dee71e7d99..4926ce61a27512c66377da9a5a4f349947d49314 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3791,6 +3791,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
