diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index ce50d014e27b3322cbe907253c4172a0afb698db..dc1492a38eb5804906778ce55b80f9f1caf9c92a 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2796,6 +2796,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
